python/mujincontrollerclient/controllerclientbase.py

```diff
@@ -1108,3 +1118,29 @@ def Restore(self, f, restoreconfig=True, restoremedia=True, timeout=600):
             except Exception as e:
                 log.exception('failed to restore: %s', e)
         raise ControllerClientError(response.content.decode('utf-8'))
+
+    #
+    # Debugging related
+    #
+
+    def ListDebugPk(self, timeout=5):
```

Comment:
I agree with @cielavenir, please don't call it `DebugPk`.

PK means primary key. For example, you can have `Scene` objects, and each `Scene` object will have a `pk` field. It won't make sense to call `Scene` objects `ScenePk`.

I prefer to call it `DebugResource`. That's how it's named on webstack. Each `DebugResource` can have a `pk` to identify itself.

So `ListDebugResources` and `DownloadDebugResource`

Reply:
Actually to make the name consistent with the other apis in this file. `ListDebugResources` -> `GetDebugResources`

And please also note that you need the objects wrapper `return self.ObjectsWrapper(self._webclient.APICall('GET', u'scene/', fields=fields, timeout=timeout, params=params))`